src/test/java/io/dashboardhub/pipelinedashboard/definitions/ProjectDefinitions.java

```diff
@@ -3,10 +3,38 @@
 import cucumber.api.java.en.Then;
 import cucumber.api.java.en.When;
 import io.dashboardhub.pipelinedashboard.PipelinedashboardApplicationTests;
+import io.dashboardhub.pipelinedashboard.domain.Project;
+import io.dashboardhub.pipelinedashboard.domain.User;
+import io.dashboardhub.pipelinedashboard.repository.ProjectRepository;
+import io.dashboardhub.pipelinedashboard.repository.UserRepository;
 import org.junit.Assert;
+import org.springframework.beans.factory.annotation.Autowired;
 
 public class ProjectDefinitions extends PipelinedashboardApplicationTests {
```

Comment:
This class is seeding the database with dummy data for the Scenario.

No reason why we can't use it for Integration tests too? We do this on other projects.